src/pages/studyView/table/CNAGenesTable.tsx

```diff
@@ -51,16 +55,34 @@ export class CNAGenesTable extends React.Component<ICNAGenesTablePros, {}> {
                                 [
                                     {
                                         name: 'Gene',
-                                        render: (data: CopyNumberCountByGene) => <span>{data.hugoGeneSymbol}</span>
+                                        render: (data: CopyNumberCountByGene) => {
+                                            const overlay = () => <span>{`Click ${data.hugoGeneSymbol} to ${_.includes(this.props.selectedGenes, data.hugoGeneSymbol) ? 'remove' : 'add'} from your query`}</span>;
+                                            return (
+                                                <DefaultTooltip
+                                                    placement="left"
+                                                    overlay={overlay}
+                                                    destroyTooltipOnHide={true}
+                                                >
+                                                    <span
+                                                        className={classnames(styles.geneSymbol, _.includes(this.props.selectedGenes, data.hugoGeneSymbol) ? styles.selectd : undefined)}
+                                                        onClick={() => this.props.onGeneSelect(data.hugoGeneSymbol)}>
```

Comment:
see comment above about anonymous functions as props. I think its probably not worth the effort here but just wanted to point it out